src/Resources/config/gaufrette.php

```diff
@@ -75,4 +76,10 @@
             ->args([[]])
 
         ->set('sonata.media.metadata.noop', NoopMetadataBuilder::class);
+
+    if (class_exists(SimpleS3Client::class)) {
```

Comment:
Is there an error without the class exist check ? Because we might lack of a class exist check for the non async S3Client then

Reply:
You are right